docs/components/badge/Example.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { wBadge } from '@warp-ds/vue';
+
+const badgeVariants = ['neutral', 'info', 'positive', 'warning', 'negative', 'disabled', 'notification', 'price',];
+const badgePositions = ['top-left', 'top-right', 'bottom-left', 'bottom-right',];
+</script>
+
+<template>
+  <div class="component space-y-24">
+    <div>
+      <h3 class="h4">Default</h3>
+      <w-badge>Rendered as neutral variant as default</w-badge>
+    </div>
+    <div>
+      <h3 class="h4">Variants</h3>
+      <div class="space-x-8">
+        <w-badge v-for="variant in badgeVariants" :key="variant" :variant="variant">{{ variant }}</w-badge>
+      </div>
+    </div>
+    <div>
+      <h3 class="h4">Positioning</h3>
+      <div class="relative border border-0 rounded-8 overflow-hidden h-96">
```

Comment:
I haven't tested these changes locally but it's possible that some of these classes need to be added to `config.js` to `docsClasses` array. Otherwise I guess they wouldn't be picked up by unocss. At least that was our previous experience :D